openziti-init.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM openziti/ziti-cli:0.33.1
+
+COPY ./openziti-init-entrypoint.sh /entrypoint.sh
+
+ENTRYPOINT [ "/entrypoint.sh" ]
+ 
```

Comment:
We don't need a dockerfile to specify the entry point. This can be specified in the compose file.

Reply:
it exists to copy the script into the container but maybe you know of a better way to accomplish that? or i could bind mount the script, but figured this was as good?

Reply:
Where is this Dockerfile being used?

Reply:
OK, I see how you are using this. I like it.!